pkg/registry/apis/provisioning/register.go

```diff
@@ -361,6 +377,73 @@ func (b *ProvisioningAPIBuilder) GetOpenAPIDefinitions() common.GetOpenAPIDefini
 	return provisioning.GetOpenAPIDefinitions
 }
 
+func (b *ProvisioningAPIBuilder) GetAPIRoutes() *builder.APIRoutes {
+	defs := b.GetOpenAPIDefinitions()(func(path string) spec.Ref { return spec.Ref{} })
+
+	statsResult := defs["github.com/grafana/grafana/pkg/apis/provisioning/v0alpha1.ResourceStats"].Schema
+
+	return &builder.APIRoutes{
+		Namespace: []builder.APIRouteHandler{
+			{
+				Path: "stats",
+				Spec: &spec3.PathProps{
+					Get: &spec3.Operation{
+						OperationProps: spec3.OperationProps{
+							OperationId: "getResourceStats", // used for RTK client
```

Comment:
Nice, didn't know we could set those manually.

Reply:
this function is *not* managed by k8s, so we have to set it manually